mesa/visualization/modules/simple_continuous_canvas.js

```diff
@@ -0,0 +1,332 @@
+var ContinuousVisualization = function(height, width, context, background_src=null) {
```

Comment:
Need to use `const`/`let` instead of `var` in all of the `var` usage in this file. You case see how to do it in #1246.

Reply:
The entire file needs to be formatted with [`prettier`](https://prettier.io/).